yield_plugin/src/yield_plugin.cpp

```diff
@@ -248,11 +277,13 @@ namespace yield_plugin
     {
       if (timesteps_since_last_req_ < config_.acceptable_passed_timesteps)
       {
+        ROS_DEBUG_STREAM("Yield for CLC");
         yield_trajectory = update_traj_for_cooperative_behavior(original_trajectory, req.vehicle_state.longitudinal_vel);
-        timesteps_since_last_req_++;
+        // timesteps_since_last_req_++;
```

Comment:
So are we never expiring the yield plan if we're not incrementing this?

Reply:
The log statement here should probably also note that we haven't received an updated negotiation this timestep, so it's obvious in post-analysis.

Reply:
removed to test its impact. It is back now.